modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -927,9 +927,14 @@ func (suite *KeeperTestSuite) TestWriteUpgradeOpenChannel() {
 	upgrade, found := suite.chainA.GetSimApp().IBCKeeper.ChannelKeeper.GetUpgrade(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
 	suite.Require().Equal(types.Upgrade{}, upgrade)
 	suite.Require().False(found)
+
 	lastPacketSequence, found := suite.chainA.GetSimApp().IBCKeeper.ChannelKeeper.GetCounterpartyLastPacketSequence(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
 	suite.Require().Equal(uint64(0), lastPacketSequence)
 	suite.Require().False(found)
+
```

Comment:
this is used in two places, could eventually add an `assertDeletedInfo` helper function to encapsulate this.